### PR TITLE
feature: add docs for extra fields

### LINKED ISSUE #11810: Add docs for: [Ldap] Add users extraFields in ldap component
| Q            | A
| ------------ | ---
| Feature PR   | symfony/symfony#31532
| PR author(s) | @Simperfit

### PR DESCRIPTION
**Pull request by Simperfit**
Fix #11810

### CODE REVIEW

File: security/ldap.rst [hunk 1]
```diff
@@ -268,6 +270,14 @@ implementation. Commonly used values are:
 If you pass ``null`` as the value of this option, the default UID key is used
 ``sAMAccountName``.
 
+extra_fields
+............
+
+**type**: ``array`` **default**: ``null``
+
+This is the custom fields to pull from the LDAP server.
+
```
Comment (OskarStark) [inline security/ldap.rst]: Maybe a note on how it behaves if an extra field isn’t available? Will it be null or does it throw an exception?

File: security/ldap.rst [hunk 2]
```diff
@@ -268,6 +270,15 @@ implementation. Commonly used values are:
 If you pass ``null`` as the value of this option, the default UID key is used
 ``sAMAccountName``.
 
+extra_fields
+............
+
+**type**: ``array`` **default**: ``null``
+
+This is the custom fields to pull from the LDAP server. If the fields does not
```
Comment (weaverryan) [inline security/ldap.rst]: Defines the custom fields...

... If any field does not exist, an Invalid... will be thrown.

**Review by weaverryan — CHANGES_REQUESTED** [review]

**Comment by javiereguiluz** [comment]
Thanks Hamza for this contribution! I made Ryan's suggested changes while merging and I also added the missing "versionadded:: 4.4" directive.

**Comment by Simperfit** [comment]
Thanks Javier, I forgot about Ryan's suggestion.

**Pull request merged**